New-HostProcessBaseImage.ps1

```diff
@@ -15,7 +15,11 @@ Copy-item -Path "cc0-legalcode.txt" -Destination "build\layer\Files\cc0-legalcod
 
 # Create layer.tar
 Push-Location build\layer
-tar.exe -cf layer.tar Files
+if ($IsLinux) {
```

Comment:
discussed something like `tarexe = if linux then tar elseif windows tar.exe` but the file slashes still needed to be flipped.  We determined that this file is small and unlikely to change much so this is good for now.